app/collins/controllers/actions/asset/PowerStatusAction.scala

```diff
@@ -31,9 +31,12 @@ case class PowerStatusAction(
   }
   override protected def onSuccess(s: CommandResult): ResponseData = {
     logSuccessfulPowerEvent()
-    val status = s.stdout.contains("on") match {
-      case true => "on"
-      case false => "off"
+    val result = s.stdout
+    val status = "error"
+    if (result.contains("Chassis Power is on")) {
+      val status = "on"
+    } else if (result.contains("Chassis Power is off")) {
+      val status = "off"
```

Comment:
This can be reduced to

```scala
val status = s.stdout match {
  case x if x. contains("Chassis Power is on") => "on"
  case x if x. contains("Chassis Power is off") => "off"
  case _ => "error"
}
```

possibly farther but my scala is not great.